### ISSUE TITLE
Enum Not Assignable

### ISSUE THREAD

**Issue by bavellone** [edgedb/edgedb-js#259]
Using the latest v0.19.5 with the following minimal schema:
```
module default {
  scalar type Env extending enum<prod, staging>;

  type App {
    required property env -> Env;
  }
}
```

I'm getting this typing error on the `env` property when trying to insert:
```
const app = e.insert(e.App, {
  env: e.Env('prod')
})

(property) env: {
    __element__: never;
    __cardinality__: Cardinality.One;
}
Type '$expr_Literal<$Env>' is not assignable to type '{ __element__: never; __cardinality__: Cardinality.One; }'.
  Types of property '__element__' are incompatible.
    Type '$Env' is not assignable to type 'never'.
```

**Comment by colinhacks** [comment]
Oops, good catch. Fix pending: https://github.com/edgedb/edgedb-js/pull/262

**Comment by colinhacks** [comment]
@bavellone the fix has landed in 0.19.6.

**Comment by bavellone** [comment]
Thanks for the super quick turnaround @colinhacks! Updated to 0.19.6 but I still seem to be getting the same error after regenerating types.
